entrypoint.sh

```diff
@@ -38,7 +40,9 @@ fi
 # This possibly changes the log directory that might be mounted in
 if check_set "${ENABLE_AIRFLOW_CHOWN}"; then
   echo "Chowning ${AIRFLOW_HOME} to ${AIRFLOW_USER}:${AIRFLOW_GROUP}..."
-  chown "${AIRFLOW_USER}:${AIRFLOW_GROUP}" -R "${AIRFLOW_HOME}/"
+  mkdir -pv "${AIRFLOW_HOME}/dags"; \
+  mkdir -pv "${AIRFLOW_HOME}/logs"; \
+  chown -R "${AIRFLOW_USER}:root" "${AIRFLOW_HOME}/"
```

Comment:
Note: We using `root` for group, to follow the official Dockerfile to cater to OpenShift standards:
https://github.com/apache/airflow/pull/9545